tools/protoc-api/src/main/proto/spine/tools/protoc/config.proto

```diff
@@ -13,29 +13,121 @@ option (internal_all) = true;
 
 // Interface to mark UUID messages.
 //
-// An empty value signifies that the messages should not be marked.
+// An interface name signifies that the additional methods should not be generated.
 //
 message UuidInterface {
 
+    // A fully-qualified name of the message interface class.
     string interface_name = 1;
 }
 
-// Interface to mark messages defined in a file with a certain name postfix.
+// Interface to mark messages defined in a file with a certain pattern.
 //
-message GeneratedInterface {
+// An interface name signifies that the additional methods should not be generated.
+//
+message GenerateInterface {
 
+    // A fully-qualified name of the message interface class.
     string interface_name = 1;
 
-    string file_postfix = 2 [(required) = true];
+    // A file pattern that defines which message types should implement the interface.
+    FilePattern pattern = 2;
 }
 
-// Interface to mark Enrichment messages.
+// Method factory that creates additional methods for UUID messages.
 //
-// An empty value signifies that the messages should not be marked.
+// An empty value signifies that the additional methods should not be generated.
 //
-message EnrichmentInterface {
+message UuidMethod {
 
-    string interface_name = 1;
+    // A fully-qualified name of the message factory class.
+    string factory_name = 1;
+}
+
+// Method factory that creates additional methods for a certain message(s).
+//
+// An empty factory name signifies that the additional methods should not be generated.
+//
+message GenerateMethod {
```

Comment:
Let's review the documentation and probably the naming of the messages in this file. At the moment it's a bit far from comprehension.